src/CalculatorUITestFramework/StandardAoTCalculatorPage.cs

```diff
@@ -0,0 +1,201 @@
+            string sourceF = this.session.PageSource;
+            if (sourceF.Contains("ExitAlwaysOnTopButton"))
+            {
+                this.ExitAlwaysOnTopButton.Click();
+                Assert.AreEqual("Standard", this.GlobalCalculatorUI.GetCalculatorHeaderText());
+            }
+            else
+            {
+                string source2 = this.session.PageSource;
+                if (source2.Contains("NormalAlwaysOnTopButton"))
+                {
+                    return;
+                }
+                else
+                {
+                    throw new NotFoundException("Could not find the 'Keep on top' button");
+                }
+            }
+        }
+        ///// <summary>
+        ///// Ensures the calculator is in AoT(Keep on top) mode, and verifies that the AoT mode through the absence of the header
+        ///// </summary>
+        public void StandardAoTMode()
+        {
+            string sourceG = this.session.PageSource;
+            if (sourceG.Contains("NormalAlwaysOnTopButton"))
+            {
+                this.EnterAlwaysOnTopButton.Click();
+                this.ExitAlwaysOnTopButton.WaitForDisplayed();
+                string sourceH = WinAppDriver.Instance.CalculatorSession.PageSource;
+                if (sourceH.Contains("Header"))
+                {
+                    throw new NotFoundException("Failed to enter 'Keep on top' mode; In AoT mode, Calculator does not have header");
+                }
+            }
+        }
+        /// <summary>
+        /// Checks to see if the size of the Calculator window is within the AoT arange of size
+        /// </summary>
+        public void AoTWindowSizeWithinRange()
+        {
+            Size getCalculatorWindowSize = WinAppDriver.Instance.CalculatorSession.Manage().Window.Size;
+            int Width = getCalculatorWindowSize.Width;
+            int Height = getCalculatorWindowSize.Height;
+            if ((Width >= 168) && (Width <= 502) && (Height >= 320) && (Height <= 502))
+            {
+                return;
+            }
+            else
+            {
+                throw new Exception("'Keep on top' window size is too small/large");
+            }
+        }
+        ///// <summary>
+        ///// Gets the AoT ToolTip text
+        ///// </summary>
+        public string GetAoTToolTipText()
+        {
+            string sourceH = this.session.PageSource;
+            if ((sourceH.Contains("Keep on top")) || (sourceH.Contains("Back to full view")))
+            {
+                if (sourceH.Contains("Keep on top"))
+                {
+                    Actions moveToAoTButton = new Actions(WinAppDriver.Instance.CalculatorSession);
+                    moveToAoTButton.MoveToElement(EnterAlwaysOnTopButton);
+                    moveToAoTButton.Perform();
+                }
+                else
+                {
+                    Actions moveToBTFVButton = new Actions(WinAppDriver.Instance.CalculatorSession);
```

Comment:
BTFV isn't a common acronym, add a comment or adjust the name.

Reply:
Change name of action to "moveToBackToFullViewVButton."  This mdification will be in my next commit